lib/src/conflicts.rs

```diff
@@ -69,6 +69,7 @@ const CONFLICT_PLUS_LINE_CHAR: u8 = CONFLICT_PLUS_LINE[0];
 static CONFLICT_MARKER_REGEX: once_cell::sync::Lazy<Regex> = once_cell::sync::Lazy::new(|| {
     RegexBuilder::new(r"^(<{7}|>{7}|%{7}|\-{7}|\+{7})( .*)?$")
         .multi_line(true)
+        .crlf(true)
```

Comment:
nit: might be safer to trim ascii whitespace at end. `ByteSlice::trim_end_with()` can be used for that purpose (because `.trim_ascii_end()` requires rustc 1.80.) 
https://docs.rs/bstr/latest/bstr/trait.ByteSlice.html#method.trim_end_with

It might also make sense to rewrite the regex with simple bytes functions. It's harder to spot edge cases of regex pattern.

Reply:
Ok, makes sense to me. I went with your first suggestion, since it seemed like the smaller change.